themes/classic/templates/_partials/breadcrumb.tpl

```diff
@@ -23,19 +23,19 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  *}
 <nav data-depth="{$breadcrumb.count}" class="breadcrumb hidden-sm-down">
-  <ol itemscope itemtype="https://schema.org/BreadcrumbList">
+  <ol>
     {block name='breadcrumb'}
       {foreach from=$breadcrumb.links item=path name=breadcrumb}
         {block name='breadcrumb_item'}
           {if not $smarty.foreach.breadcrumb.last}
-            <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
-              <a itemprop="item" href="{$path.url}"><span itemprop="name">{$path.title}</span></a>
-              <meta itemprop="position" content="{$smarty.foreach.breadcrumb.iteration}">
+            <li>
+              <a href="{$path.url}"><span>{$path.title}</span></a>
+              <meta content="{$smarty.foreach.breadcrumb.iteration}">
             </li>
           {elseif isset($path.title)}
-            <li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
-              <span itemprop="name">{$path.title}</span>
-              <meta itemprop="position" content="{{$smarty.foreach.breadcrumb.iteration}}">
+            <li>
+              <span>{$path.title}</span>
+              <meta content="{{$smarty.foreach.breadcrumb.iteration}}">
```

Comment:
typo

Reply:
where ?

Reply:
{{$smarty.foreach.breadcrumb.iteration}} double brackets, but its not your bug 👍

Reply:
Corrected, yes strange because I don't have the same typo in 1.7.7.x branch  apparently only on develop.